test/format/config.c

```diff
@@ -929,6 +929,8 @@ config_transaction(void)
             testutil_die(EINVAL, "timestamps require snapshot isolation");
         if (g.c_txn_freq != 100 && config_is_perm("transaction.frequency"))
             testutil_die(EINVAL, "timestamps require transaction frequency set to 100");
+        if (g.c_rebalance && config_is_perm("ops.rebalance"))
```

Comment:
This deserves a comment - I'm also surprised it doesn't more closely match the code here:
https://github.com/wiredtiger/wiredtiger/blame/develop/test/format/config.c#L632

Reply:
The previous change was made there but it wan't working as expected. I'm not familiar with configuration of test format and I'm not sure why.

Reply:
This change doesn't do anything, except return an error if the test/format user attempted to explicitly configure rebalance (that's what "perm" is) and timestamps.

i.e: if the random configuration chooses rebalance and timestamps, it will still try to run rebalance. I don't see how this can fix the problem.

Reply:
OK, so I should change the random configuration as well.